editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -1647,6 +1647,9 @@ void SpatialEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 							Transform t;
 							Vector3 local_scale;
 
+							float click_distance = click.distance_to(_edit.center);
+							motion = ((motion / original.basis.get_scale()) / (Vector3(click_distance, click_distance, click_distance) / original.basis.get_scale()));
```

Comment:
The divisions by `original.basis.get_scale()` are redundant since they happen on both sides of the larger division. The code can be simplified down to this:
```suggestion
							motion /= click.distance_to(_edit.center);
```